src/i18n/translations/vn.yml

```diff
@@ -33,11 +33,27 @@ recover.error: "Liên kết đã hết hạn! Xin vui lòng bắt đầu lại t
 testimonials.title: "Testimonials"
 testimonials.loadmore: "Load More"
 # BALANCE
-balance.available: "Khả dụng"
-balance.invested: "Đã Đầu Tư"
-balance.profit: "Lợi Nhuận Chưa Thực Hiện"
-balance.total: "Tổng vốn sở hữu"
-balance.profitlosses: "UP / L #"
+balance.available: "Available"
```

Comment:
Any reason the other translation files have been updated to contain english content?

Reply:
well let me remember,  actually this was missing from here, so I just added here.